### DISCUSSION TITLE
<title>mars 2020 badge removed form my profile

### DISCUSSION THREAD

**Discussion by atrebbi** [community/community#61286] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why was the mars 2020 badge removed from my profile ?
I mean this: https://github.com/readme/featured/nasa-ingenuity-helicopter
was it removed for everyone or only from my profile ? and why ?

**Comment by GhostOf0days** [comment]
Could you please make sure you didn't accidentally [hide](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#hiding-an-individual-achievement-on-your-profile) it?

**Reply by atrebbi** [reply]
yes, thanks, it was hidden but still present ; maybe I accidentally hided it

**Reply by GhostOf0days** [reply]
No problem.
